wolfssl/certs_test.h

```diff
@@ -6871,5 +6991,31 @@ static const int sizeof_client_ed25519_key = sizeof(client_ed25519_key);
 
 #endif /* HAVE_ED25519 */
 
+#if defined(HAVE_CURVE25519)
```

Comment:
Same as comment above, let's put this in a check for `USE_CERT_BUFFERS_25519` in addition to `HAVE_CURVE25519`.